geowebcache/core/src/main/java/org/geowebcache/filter/security/SecurityDispatcher.java

```diff
@@ -73,6 +73,14 @@ public void checkSecurity(TileLayer layer, @Nullable BoundingBox extent, @Nullab
         }
     }
 
+    /**
```

Comment:
As far as I know, if GWC is deployed stand-alone, there are indeed no SecurityFilter, the only implementation I'm aware of is in GeoServer (which you have updated accordingly [in the GS counterpart](https://github.com/geoserver/geoserver/pull/8189)).

I'm asking because I don't want to give the wrong impression in the GWC release notes... the commit message should probably be something along the lines of "Support hiding GWC version when integrated in GeoServer".
(we use the commit messages to build the GWC release notes)